SpatialGDK/Source/SpatialGDK/Private/Interop/CrossServerRPCSender.cpp

```diff
@@ -24,8 +28,20 @@ void CrossServerRPCSender::SendCommand(const FUnrealObjectRef InTargetObjectRef,
 
 	CommandRequest CommandRequest(SpatialConstants::SERVER_TO_SERVER_COMMAND_ENDPOINT_COMPONENT_ID,
 								  SpatialConstants::UNREAL_RPC_ENDPOINT_COMMAND_ID);
-	RPCPayload::WriteToSchemaObject(CommandRequest.GetRequestObject(), InTargetObjectRef.Offset, Info.Index, FMath::RandHelper(INT_MAX),
+
+	uint64 UniqueRPCId = FMath::RandHelper(INT_MAX);
```

Comment:
Curiosity, was 64bit chosen because it has enough chance of not colliding and not affecting egress, versus let's say a 128 bit GUID?

Reply:
Good question, it's not even 64-bit because INT_MAX is often 32-bit and how random is FRand (internally rand()) for uniqueness? What if it was seeded poorly?

Reply:
Mmmm. Seems like a weakness in our system for a random crash

Reply:
Yeah I wonder if a better approach might be N-bits of worker ID and a counter?